content/blog/2019-02-18-stats19.md

```diff
@@ -0,0 +1,1014 @@
+[STATS19](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/775149/Operational_Metrics_Manual.pdf)
+(note the capital letters). There is another document called [STATS20](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/230596/stats20-2011.pdf), which is the guidance for officers filling in a STATS19 form. 
+
+An important point is that the dataset omits crashes in which
+nobody was hurt, as emphasised by another government
+[document](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/743853/reported-road-casualties-gb-notes-definitions.pdf):
+
+> The STATS19 data are therefore not a complete record of all injury
+> accidents and this should be borne in mind when using and analysing
+> the data. However, they remain the most detailed, complete and
+> reliable single source of information on road casualties covering the
+> whole of Great Britain, in particular for monitoring trends over time
+
+The Department for Transport (DfT) also names the dataset [STATS19 on
+the main web page that links to open access road crash
+data](https://data.gov.uk/dataset/cb7ae6f0-4be6-4935-9277-47e5ce24a11f/road-safety-data).
+
+The importance of road safety and informed decision making based on
+crash data cannot be overstated. Deliberately avoiding the matter of
+life and death of road safety, two numbers from a strategy
+[document](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/8146/strategicframework.pdf)
+by the UK government (2011) are worth mentioning to show the scale of
+the numbers:
+
+> The economic welfare costs \[of road collisions\] are estimated at
+> around £16 billion a year while insurance payouts for motoring claims
+> alone are now over £12 billion a year.
+
+Even more shocking are the [global
+statistics](https://www.who.int/gho/road_safety/mortality/en/), as
+summarised by an open access and reproducible academic [paper that uses
+data from the package to explore car-pedestrian
+crashes](https://github.com/Robinlovelace/stats19-gisruk):
+
+> While so many people die on the roads each year in the UK (1,793
+> people in 2017, 3 deaths per 100,000) and worldwide (1,250,000 people
+> in 2015, 17 deaths per 100,000) and ‘vision zero’ remains a Swedish
+> dream (Johansson 2009), we urge people researching STATS19 and other
+> road safety datasets to focus on a more urgent question: how to stop
+> this carnage?
+
+#### The road crash data in stats19
+
+There are three main different types of CSV files released by the DfT:
+`accidents`, `vehicles` and `casualties` tables. There is a schema
+covering these tables but a good amount of work is needed to understand
+it, let alone be able to the data contained within the files and convert
+the integers they contain into meaningful data.
+
+The annual statistics have not been released in a consistent way either,
```

Comment:
Removed the starting sentence, but I see more value in describing what "STATS19" includes so that we don't just dive into the code and make it a "blackbox". See if you still think it needs further condensations.